docs/api/account.rst

```diff
@@ -1,25 +1,27 @@
 Account
 =======
 
----------
-Interface
----------
+---------------------
+BaseAccount interface
+---------------------
 
 .. py:module:: starknet_py.net.account.base_account
 
 .. autoclass:: BaseAccount
     :members:
-    :member-order: bysource
+    :undoc-members:
```

Comment:
Why was this added?

Reply:
To include methods (or properties) without a docstrings

Reply:
I'd prefer if we only included methods (and properties) with docstrings and removed `undoc-members` from the config.